src/main/java/org/maxandersen/starfish/Starfish.java

```diff
@@ -72,25 +72,26 @@ public int run(String... args) throws Exception {
     
 
 
-    //Clonnning Git Repo
-    //Expected parameter: https://github.com/user-name/repo-name.git
+    
     String repo_name=args[0].substring(args[0].lastIndexOf("/"),args[0].lastIndexOf(".")); //Extracts the Name of Repository
-    //String clone_path= "/home/fahad/MyProjects/starfish_clonned/"; 
+   
     String originUrl = args[0];
     Path directory = Paths.get(clone_path+repo_name);
+
     if(!Files.exists(directory)) //Check if the user cloned the repo previously and in that case no cloning is needed
-    gitClone(directory, originUrl);
+    gitClone(directory, originUrl);//Calling function to clone the repository
```

Comment:
that comment doesn't say anything beyond what the method name says. 
```suggestion
    gitClone(directory, originUrl);
```